__test__/app.test.js

```diff
@@ -9,18 +9,37 @@ const app = require('../app');
 describe('/api', () => {
   afterAll(() => connection.destroy());
   beforeEach(() => connection.seed.run());
-
-  // Ayako BRANCH OUT FOR EACH REQUEST!! DON'T WORK ON MASTER
-  describe('/food-items', () => {
-    it('GET 200 - responds with an array of all food_items', () => {
+// Ayako BRANCH OUT FOR EACH REQUEST!! DON'T WORK ON MASTER 
+  describe('GET /api/food-items', () => {
+    it('SUCCESS status 200 - responds with an array of all food_items', () => {
       return request(app)
-        .get('/api/food-items')
-        .expect(200)
-        .then(({ body }) => {
-          expect(body.foodItems).toHaveLength(21);
-        });
-    });
-  });
+      .get('/api/food-items')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.foodItems).toHaveLength(21)
+      })
+    })
+  })
+  describe('POST /api/food-items', () => {
```

Comment:
should also include a body missing required fields as all fields are not nullable

Reply:
I will add an error handling for that one.